build-system/conformance-config.textproto

```diff
@@ -168,6 +168,22 @@ requirement: {
 }
 
 requirement: {
+  type: BANNED_PROPERTY_READ
```

Comment:
I see. That is a bit of an ambiguous case. You only strictly need to taint such an object if it cross compilation units. Not really related to this PR.